com.jetbrains.IntelliJ-IDEA-Community.json

```diff
@@ -1,22 +1,48 @@
 {
   "app-id": "com.jetbrains.IntelliJ-IDEA-Community",
   "command": "idea",
-  "runtime": "org.freedesktop.Sdk",
-  "runtime-version": "18.08",
+  "runtime": "org.gnome.Platform",
```

Comment:
This is not needed. I understand `libsecret` is already included in GNOME, but as you might guess, there are also tons of other libraries that are not needed; I like to keep things to a minimum, and so far I don't see any errors in the system logs for this Flatpak – except the `ln` to `/etc/mtab` but I'm aware of that and it's harmful.

And by the way, the Flatpak is agnostic (to a great extent) to the host operating system, so it doesn't really matter what the user have installed; sometimes some "tweaks" are needed, but overall, the _experience_ should be seamlessly.